### PR TITLE
Fix/pvalue nullcrash

### PR DESCRIPTION
**Pull request by alex-wilmer**
- fix pvalue and survival null crashes when buckets return null.
- pass initial value to InputWithWarning

### CODE REVIEW

File: src/packages/@ncigdc/modern_components/CohortComparison/CohortComparison.js [hunk 1]
```diff
@@ -176,9 +176,7 @@ export default compose(
             palette={[set1_colour, set2_colour]}
             style={{ flex: 1, width: 100 }}
           />
-          <div
-            style={{ marginLeft: 50, flex: 1, width: 100, textAlign: 'center' }}
-          >
+          <div style={{ marginLeft: 50, flex: 1, width: 100 }}>
```
Comment (Jephuff) [inline src/packages/@ncigdc/modern_components/CohortComparison/CohortComparison.js]: the link should probably still be centered
image

Comment (alex-wilmer) [reply src/packages/@ncigdc/modern_components/CohortComparison/CohortComparison.js]: how about this instead:

image

Comment (Jephuff) [reply src/packages/@ncigdc/modern_components/CohortComparison/CohortComparison.js]: works for me 👍

**Comment by Jephuff** [comment]
Could you make the link inline-block? Wraps weird right now 
image

**Pull request merged**